bundles/org.openhab.binding.kostalinverter/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -856,16 +856,9 @@
 		<state readOnly="false"/>
 	</channel-type>
 	<channel-type id="device-local-smart-battery-control">
-		<item-type>Switch</item-type>
+		<item-type>String</item-type>
```

Comment:
Great that you received feedback from the user! It would make sense to make this a Switch if the value is true/false, although if the early adopter needs to edit his configuration.